### ISSUE TITLE
Iframe / frame doesn't work

### ISSUE THREAD

**Issue by lkclkc2003** [mozilla/thimble.mozilla.org#2590]
As titled

**Comment by humphd** [comment]
@lkclkc2003 can you give a bit more info?  Are you adding an `iframe` to your project and it's failing?  What does "doesn't work" mean in this context?  If you can give some more info, we can try to help or fix.

**Comment by lkclkc2003** [comment]
@humphd Yes. When I am trying to add <frame> or <iframe> it just shows nothing. Thx

**Comment by humphd** [comment]
Works for me, see https://thimble.mozilla.org/en-US/projects/425533/remix

You need to give me more specifics.  I believe you when you say it doesn't work for you, but in order to debug this, I need info (e.g., browser, HTML you are using, OS, etc).
